lib/adapters/autocomplete-adapter.ts

```diff
@@ -296,7 +296,7 @@ export default class AutocompleteAdapter {
   //
   // Returns a {String} containing the AutoComplete+ suggestion type equivalent
   // to the given completion kind.
-  public static completionKindToSuggestionType(kind: number | null): string {
+  public static completionKindToSuggestionType(kind: number | undefined): string {
```

Comment:
`CompletionItem.kind` uses `number | undefined` (or more accurately, a bunch of specific numbers unioned with `undefined`